components/ProfilePage/PreWalletCollection.tsx

```diff
@@ -187,10 +157,18 @@ const PreWalletCollection = () => {
                   overflow-y-auto overflow-x-hidden
                   pr-2"
           >
-            {mockData.map((data, i) => (
+            {ownedNfts.map((data, i) => (
               // eslint-disable-next-line react/no-array-index-key
               <div key={i} className="flex flex-col items-center gap-y-[5px]">
-                <div className="w-[30px] h-[30px] samsungS8:w-[35px] samsungS8:h-[35px] lg:w-[93px] lg:h-[93px] bg-white rounded-[5px] lg:rounded-[15px]" />
+                {data.image ? (
+                  <img
+                    className="w-[30px] h-[30px] samsungS8:w-[35px] samsungS8:h-[35px] lg:w-[93px] lg:h-[93px] bg-white rounded-[5px] lg:rounded-[15px]"
+                    src={data.image}
+                    alt={data.label}
+                  />
+                ) : (
+                  <div className="w-[30px] h-[30px] samsungS8:w-[35px] samsungS8:h-[35px] lg:w-[93px] lg:h-[93px] bg-white rounded-[5px] lg:rounded-[15px]" />
+                )}{" "}
```

Comment:
{" "} => Can I remove this? 🚮

Reply:
should be fixed now 👍

Reply:
thank you, gonna leave this in your capable hands 🙇